src/Responsive.php

```diff
@@ -141,7 +145,11 @@ protected static function getGlideUrl(Asset $asset, string $preset, string $fit,
             return URL::encode($url);
         }
 
-        GenerateGlideImageJob::dispatch($asset, $preset, $fit, $format);
+        if (config('queue.default') === 'redis') {
```

Comment:
I think this should listen to `justbetter.glide-directive.default_queue`? 

But I don't think having a vendor prefix directory is a common thing to do, eg: `justbetter/glide-directive.php` => `glide-directive.php`, or maybe better: `statamic/glide-directive.php`. Not sure what other Statamic addons do?